autoload/rhubarb.vim

```diff
@@ -323,6 +325,9 @@ function! rhubarb#FugitiveUrl(...) abort
   elseif get(opts, 'type', '') ==# 'blob' || opts.path =~# '[^/]$'
     let escaped_commit = substitute(commit, '#', '%23', 'g')
     let url = root . '/blob/' . escaped_commit . '/' . path
+    if get(opts, 'line1') > 0 && index(s:github_markup_types, get(split(bufname(), '\.'), -1, '')) >= 0
```

Comment:
Use `fnamemodify(path, ':e')` to get the extension.

Reply:
Done